src/main/kotlin/com/strumenta/kolasu/model/Processing.kt

```diff
@@ -145,6 +145,7 @@ fun Node.find(predicate: (Node) -> Boolean): Node? {
 /**
  * Recursively execute "operation" on this node, and all nodes below this node that extend klass.
  */
+@Suppress("UNCHECKED_CAST")
```

Comment:
Should we add a comment to explain why this is ok here?

Reply:
This one probably also disappears.